front/app/containers/App/index.tsx

```diff
@@ -373,18 +373,17 @@ class App extends PureComponent<Props, State> {
             verification: !!sso_verification,
             requiresConfirmation: shouldConfirm,
             modalNoCloseSteps: ['confirmation'],
-            verificationContext: !!(
+            verificationContext:
               sso_verification &&
               sso_verification_action &&
               sso_verification_id &&
               sso_verification_type
-            )
-              ? {
-                  action: sso_verification_action as any,
-                  id: sso_verification_id as any,
-                  type: sso_verification_type as any,
-                }
-              : undefined,
+                ? {
+                    action: sso_verification_action as any,
```

Comment:
Shall we also warn against any type value ?

Reply:
I considered this but we are using `any` extensively in the codebase already and also, I think it's a bit of an overkill to be super strict about it and forbid it outright. Happy to discuss this further, if our opinions diverge.